distributed/dashboard/components/scheduler.py

```diff
@@ -377,9 +380,34 @@ def __init__(self, scheduler, width=600, **kwargs):
             self.root.toolbar_location = None
             self.root.yaxis.visible = False
 
-            hover = HoverTool()
-            hover.tooltips = "@worker: @proc_memory{0.00 b} (@width{0.00 b} @memtype)"
-            hover.point_policy = "follow_mouse"
+            hover = HoverTool(
+                point_policy="follow_mouse",
+                tooltips="""
+                            <div>
+                                <span style="font-size: 12px; font-weight: bold;">@worker:</span>&nbsp;
```

Comment:
Slightly off topic but have we considered using the worker name for this? I have the impression that a worker name is usually much easier to read than an IP

Reply:
I was wondering if we need the IP at all, I noticed that the name is just an `int` (would this not be a number ever?). I guess we could put the name, and if anyone needs the IP they can always check the Workers tab on the Dashboard.

Reply:
That's a fair point @fjetter, worker names are more readable. Though for now I have a slight preference to stick with addresses since they are used throughout the rest of the dashboard to identify workers (e.g. see the `/info/main/workers.html` dashboard page). We can certainly re-evaluate whether or not we should switch to worker names later, but for now I'd prefer to be consistent